Fixes: No scroll view in Settings Activity #592

da-nish opened the pull request:
i have added scroll view in settings activity
fixes #592

m-murad committed:
fix: disable views in view mode

Signed-off-by: m-murad <murad.kuka@gmail.com>

isabelcosta committed:
Merge pull request #86 from m-murad/refactor

Refactor update profile fragment.

isabelcosta committed:
Merge pull request #93 from haroldadmin/kotlin-1.3

chore: Update kotlin to 1.3.0

Manik1094 committed:
fix: send request activity null end date (#80)

mohak1283 committed:
fix: User can update mentor/mentee availability

isabelcosta committed:
Merge pull request #97 from mohak1283/profile

fix: User can update mentor/mentee availability

mohak1283 committed:
feat: implemented settings screen

isabelcosta committed:
Merge pull request #104 from mohak1283/settings

Implemented Settings screen

m-murad committed:
feat: bidirectional observability in EditProfileFragment databinding

Signed-off-by: m-murad <murad.kuka@gmail.com>

isabelcosta committed:
Merge pull request #109 from m-murad/fix#108

feat: bidirectional observability in EditProfileFragment databinding

baghaii committed:
fix: setting icon in a fixed place

isabelcosta committed:
Merge pull request #114 from baghaii/profile_menu

fix: setting menu icon in a fixed position

chhavi1614 committed:
feat : Back button added to settings screen

isabelcosta committed:
Merge pull request #117 from chhavi1614/develop

Add Back button added to settings screen

shiv07tiwari committed:
fix: back navigation after logout (#123)

haroldadmin committed:
fix: Add input validation to EditProfileFragment

Fixes Issue #110. This commit adds input validation to the
EditProfileFragment. An extra file called States.kt has been created to represent possible
error states. More errors can be added to
this class as the application grows and the number of required fields in
a profile increases.

isabelcosta committed:
Merge pull request #115 from haroldadmin/validation-editprofile

fix: Add input validation to EditProfileFragment

baghaii committed:
test: add LoginActivityTest

Took the test developed in PR #72 and added it to a current branch
with some slight modifications

test: add LoginActivityTest

Added comments to reduce Codacy quality review issues.

test: add LoginActivityTest

Still attempting to satisfy Codacy

test: add LoginActivityTest

Rimjhim28 committed:
feat: add new task screen

isabelcosta committed:
Merge pull request #121 from Rimjhim28/tasks_screen

Add new mentorship task screen

ananyaarun committed:
docs: added CODE_OF_CONDUCT.md file

isabelcosta committed:
Merge pull request #137 from ananyaarun/issue_128

Added CODE_OF_CONDUCT.md file

shiv07tiwari committed:
fix: update list after deleting a request (#135)

jashpreet committed:
Added Logout confirmation dialogue

Commit:
fix: Add scroll bar to Members List screen

isabelcosta committed:
Merge pull request #141 from jashpreet/develop

feat : Added Logout confirmation dialog

isabelcosta committed:
Merge pull request #143 from shaliniv16/add-scroll-bar

fix: Add scroll bar to Members List screen

Shritesh99 committed:
docs: Add Reporting Guidelines file into .github folder 


docs: Add Reporting Guidelines file into .github folder

arpi-r committed:
fix: Members List does not have an empty state

isabelcosta committed:
Merge pull request #147 from Shritesh99/issue_Reporting_Guidelines

Fixes #138: Add Reporting Guidelines file into .github folder

isabelcosta committed:
Merge pull request #155 from arpi-r/issue#89

Fixed Members List lack of empty state

ak8527 committed:
feat: Add change password in Setting Screen

ak8527 committed:
fix: Login on pressing enter in keyboard

isabelcosta committed:
Merge pull request #163 from ak8527/keyLogin

fix: Login on pressing enter in keyboard

asankova committed:
docs: replace auto-generated KotlinDocs

isabelcosta committed:
Merge pull request #159 from ak8527/changePassword

feat: Add change password in setting screen

isabelcosta committed:
Merge pull request #131 from baghaii/login_test

Add test for LoginActivity

Lavanyagaur22 committed:
feat : Created a confirmation dialog when cancelling the current relation.

isabelcosta committed:
Merge pull request #165 from gingerkirsch/issue-96-kotlin-docs

[#96] Replace auto-generated KotlinDocs

haroldadmin committed:
feat: Add user statistics to home screen

Fixes issue #20. This adds user stats to the home screen
containing numbers for relations/requests, and the most recent
achievements.

isabelcosta committed:
Merge pull request #144 from Lavanyagaur22/develop

 Created a confirmation dialog when cancelling a relation

isabelcosta committed:
Merge pull request #140 from haroldadmin/home-fragment

feat: Add user statistics to home screen

KING-SID committed:
fix: Add new line at the end of files to fix error

isabelcosta committed:
Merge pull request #191 from KING-SID/develop

fix: Add new line at the end of files to fix error

haroldadmin committed:
refactor: Migrate to AndroidX

This commit maps all existing dependency packages to their AndroidX
namespace equivalents. Redundant dependencies on v4 and v7 versions of
the support library have also been removed. BuildSrc module buildscript
has been modified to include additional required dependencies

isabelcosta committed:
Merge pull request #178 from haroldadmin/androidx

refactor: Migrate to AndroidX

haroldadmin committed:
fix: Fix travis built apk crash

isabelcosta committed:
Merge pull request #204 from haroldadmin/develop

fix: Fix travis built apk crash

mohak1283 committed:
fix: hide tabs on the current relation empty state (#134)

pranay2063 committed:
fix: crash from wrong textview package (#206)

jenpaff committed:
fix: fix empty state behaviour (#208)

removed duplicate fragment which was causing the error, remove tabs when there is no current relation

rjain90 committed:
refactor: change expiry serialized name (#213)

* refactor: change expiry serialized name

jenpaff committed:
feat: add click effect to settings options (#209)

sidvenu committed:
fix: login EditText misalignment in landscape mode (#215)

saumya1singh committed:
feat: add About activity (#219)

victorvicari committed:
feat: add back button to About screen (#228)

beccadaniel committed:
feat: Included links to terms of condition text on the register screen (#230)

* feat: Included links to terms of condition text on the register screen
* refactor: changed textview id

jenpaff committed:
feat: fetch all tasks for mentorship-relation (#231)

* feat: fetch all tasks for mentorship-relation
* create TaskService
* create TaskDataManager
* call API to fetch all tasks from DB
* small UI changes -> add new task button

rgu0011 committed:
style: Change About view title (#234)

* Changed About view title to 'About'
* fix: Removed an IDE config file from previous commit

ashar-7 committed:
fix: Enable cleartext support in debug mode (#243)

asawesome07 committed:
feat: added find members button

Added a find members button in the relations tab that when clicked navigates to the Members page and changes the tab to Members in the Bottom Navigation Tab.

Isabel-S committed:
fix: show snackbar when both users are only available to mentor

Commit:
docs: Add screenshots to README.md

docs: Add screenshots to README.md

docs: Add screenshots to README.md

daksha-aeer committed:
feat: integrate past mentorship relations from backend via API

Working
1. RelationshipService.kt makes API call to mentorship_relations/past
2. getPastRelationships() in RelationDataManager.kt reads this as Observable<List<Relationship>>
3. getPastMentorshipRelations() in RequestsViewModel.kt subscribes to the data and manages exception handling
4. getPastMentorshipRelations() called in RequestsFragment.kt. It displays this data in fragment_requests.xml

daksha-aeer committed:
feat: disable past dates in DatePickerDialog for sending mentorship requests

datePicker.minDate property added in SendRequestActivity.kt .
Reference: https://stackoverflow.com/a/57607621

daksha-aeer committed:
feat: Use GSON's field naming policy to change field names between camel and snake case

Some variables had to be renamed to match with GSON values.

Commit:
refactor: renamed TaskViewModel to TasksViewModel

Commit:
feat: Delete account function

feat: Delete account function

Commit:
fix: improve AboutActivity landscape layout

asawesome07 committed:
fix: disable send request button

disabled send request button for member profile not available to mentor/for mentoring

daksha-aeer committed:
feat: Add animations to activity transitions

Sliding animation added to activity transitions

Working:
- res/anim folder created which has styles in xml format.
- CustomActivityAnimation created in styles.xml which reads files in /anim.
- CustomActivityAnimation is utilized by AppTheme. This applies the style globally.

daksha-aeer committed:
refactor: Use Kotlin lazy {} delegated properties to initialize ViewModels

bartekpacia committed:
test: added espresso test for LoginActivity

polalab committed:
fix: Retain values on orientation change in Sign Up page (#454)

sunnyyvinay committed:
style: Center-align the Text on About Page (#397)

polalab committed:
fix: Retain Change Password Dialog in SettingsActivity (#403)

mayburgos committed:
Merge pull request #381 from rayhanhanaputra/rename_taskviewmodel

[Program:GCI] refactor: renamed TaskViewModel to TasksViewModel

mayburgos committed:
Merge pull request #296 from Isabel-S/develop

[Program:GCI] Show SnackBar when both users are 'only available to mentor'

mayburgos committed:
Merge pull request #401 from IlyaMaier/delete_account

[Program: GCI] Delete account function and some code improvements in SettingsActivity

mayburgos committed:
Merge pull request #308 from IlyaMaier/screenshots

[Program: GCI] Add screenshots to README.md file

mayburgos committed:
Merge pull request #363 from daksha-aeer/task22finalfixed

[Program:GCI] feat: Use GSON's field naming policy to change field names between camel and snake case

mayburgos committed:
Merge pull request #311 from daksha-aeer/task11

[Program: GCI] feat: Integrate past mentorship relations from backend via API

mayburgos committed:
Merge pull request #517 from daksha-aeer/lazyInitialization

[Program:GCI] Use Kotlin lazy {} initialization for ViewModels

daksha-aeer committed:
Merge branch 'gci-dev' into activityAnimations

m-murad committed:
Merge pull request #476 from daksha-aeer/activityAnimations

[Program:GCI] Add animations to activity transitions

m-murad committed:
Merge pull request #536 from bartekpacia/add_espresso_tests

[Program: GCI] Added Espresso tests for LoginActivity

m-murad committed:
Merge pull request #282 from asawesome07/findmembersbutton

[Program:GCI] Create a 'Find Members' button (Mentorship-Android)

daksha-aeer committed:
fix: app crash on orientation change while creating new task (#409)

daksha-aeer committed:
feat: Add default profile picture in fragment_edit_profile.xml (#287)

anny-wu committed:
fix: login screen overlap in landscape orientation (#489)

polalab committed:
fix: request detail screen overlap (#425)

polalab committed:
refactor: remove unnecessary parcelcreator lint suppressor (#376)

laxyapahuja committed:
fix: blank request notes check (#291)

annabauza committed:
feat: improve AboutActivity landscape layout

feat: improve AboutActivity landscape layout

annabauza committed:
feat: disable past dates in DatePickerDialog while sending mentorship requests

Buddhiprabha committed:
Merge pull request #464 from asawesome07/disableSendRequestBtn

[Program: GCI] Disable send request button if the person is not available to be mentor and mentee

annabauza committed:
fix: crashes and travis failure (#564)

polalab committed:
fix: no loosing information while rotating screen in the edit profile section (#346)

asawesome07 committed:
feat: app logs out user when JWT expires (#475)

asawesome07 committed:
Feature: Prevent a user from sending multiple mentor/mentee requests to the same person (#465)

rudrakshjhalanii07 committed:
Fix: Lint warnings in various files. (#393)

asawesome07 committed:
feat: added pending call for API (#320)

Co-authored-by: Anna Bauza <31966073+annabauza@users.noreply.github.com>

daksha-aeer committed:
feat: Add ServiceView widget to search members by name (#318)

Co-authored-by: annabauza <31966073+annabauza@users.noreply.github.com>

annabauza committed:
chore: create debug_locahost configuration (#565)

ilyamaier committed:
feat: Add filter and sort by functionality (#511)

Co-authored-by: annabauza <31966073+annabauza@users.noreply.github.com>

daksha-aeer committed:
feat: Functionality to add new tasks and achievements (#521)

Co-authored-by: annabauza <31966073+annabauza@users.noreply.github.com>

asawesome07 committed:
feat: added prominent top app bar image (#357)

Co-authored-by: Anna Bauza <31966073+annabauza@users.noreply.github.com>

daksha-aeer committed:
feat: Make change password dialog non dismissable (#366)

ooooda committed:
fix: deleting a request from all lists (#333)

AnuthaDev committed:
feat: Add a toast to confirm user exit (#258)

asawesome07 committed:
feat: add feedback screen (#268)

bartekpacia committed:
feat: add swipe-to-refresh to Home and Members (#505)

* HomeFragment
* MembersFragment

ashar-7 committed:
feat: Integrate WebView in AboutActivity (#295)

bartekpacia committed:
feat: added animations to MembersFragment (#422)

Co-authored-by: annabauza <31966073+annabauza@users.noreply.github.com>

annabauza committed:
fix: remove redundant idea files and add them to gitignore (#611)

* fix: crashes and travis failure
* fix: remove redundant idea files and add them to gitignore
* remove duplicated configuration from gradle

mayburgos committed:
Merge pull request #604 from anitab-org/gci-dev

Merging Google Code-in work into develop branch

gaurivn committed:
docs: change references from systers to anitab-org (#566)

Rita-Okonkwo committed:
docs: add license to README (#610)

mayburgos commented:
@da-nish We don't accept PRs that are opened without a Maintainer approving the issue and assigning someone to the issue. 

Please work on your GSoC application if you are applying to GSoC before you start to create issues and open PRs on projects.

I haven't seen you complete the [Student Instructions](https://github.com/anitab-org/anitab-org.github.io/wiki/GSoC-2020-Student-Instructions) or attend our [Open Sessions](https://anitab-org.zulipchat.com/#narrow/stream/224798-gsoc/topic/Open.20Sessions.20.26.20Project.20Feedback) to pitch your GSoC ideas.

Please utilize all the resources we provide so you can create a strong proposal.

isabelcosta committed:
docs: update CONTRIBUTING.md (#602)

isabelcosta committed:
fix: replace Slack references to Zulip (#597)

ilyamaier committed:
fix: blank space trimming in LoginActivity (#362)

isabelcosta committed:
fix: change backend URL to temporary one (#637)

isabelcosta committed:
chore: fix systers references generate-apks script

sonishreyas committed:
Fixed typo on LoginActivityTest (#638)

foongminwong committed:
Merge pull request #640 from isabelcosta/639-fix-generate-apk-script

Fix systers references generate-apks script

Rethran committed:
fix: Filter and Feedback parent class (#636)

Changed parentclasses from AppcompactActivity to BaseActivity
in FeedbackActivity and FilterActivity

ilyamaier committed:
feat: Add password security check (#423)

feat: Add password security check

Kavya-24 committed:
fix: Fix Typos in Feedback Page (#673)

kbhardwaj123 committed:
docs: replace reference of Slack to Zulip in README (#643)

* reference Zulip instead of slack
* README.md: make suggested changes
* fix spacing

pknotfound committed:
fix: refresh profile fragment when update profile (#692)

Kavya-24 committed:
fix: ambiguous subscript of empty username/email (#690)

pknotfound committed:
fix: register screen buttons alignment (#617)

Kavya-24 committed:
fix: registration with mentee/mentor avail unchecked (#686)

* Fixes signUp with mentee/mentor unchecked
* Add radio buttons : Mentor, Mentee, Both
* fix isAvailableForBoth

pratikghule committed:
refactor: put hardcoded Feedback screen strings resources (#630)

Kavya-24 committed:
fix: hidden last items of the Profile Fragment (#699)

mtreacy002 committed:
fix: availability mismatch on my profile and send request (#687)

devptyagi committed:
fix: app crash when clicking edit profile button before the user data was loaded (#705)

The app crashed if we clicked the edit profile button while the profile data is still being fetched. The issue is now fixed. The edit profile button will not trigger any action until the profile data has not been fetched.

annabauza committed:
chore: Update push apk from proper account (#711)

devanshi7799 committed:
feat: change splash screen background color to white (#634)

HaripriyaB committed:
fix: #679 Required error not visible in Confirm password field (#706)

devptyagi committed:
fix: #660 App crashes when anything is typed in the search bar in Members Fragment (#703)

The app crashed if we typed anything into the search bar in the members fragment. The issue is now fixed.

jddeep committed:
fix: #623 Spacing of item list on Settings Screen (#629)

DhaneshShetty committed:
fix: change Send request to Send Request (#720)

Kavya-24 committed:
fix: Save in (Profile) when no change detected (#698)

DhaneshShetty committed:
feat: back button takes to Login from SignUp screen (#721)

DhaneshShetty committed:
fix: Bottom Navigation Tab Reselected Bug (#725)

Commit:
fix: buttons size in about activity (#722)

DhaneshShetty committed:
fix: ScrollView Bug in FeedBack screen (#723)

sanchi0204 committed:
fix: new logo with transparent background (#731)

bartekpacia committed:
feat: add intro slider when signing in (#531)

* feat: added intro slider
* Update AndroidManifest.xml
* Update build.gradle

Commit:
feat: Order requests in a descending order based on the creation date (#713)

rahulshah456 committed:
feat: pagination added to members screen (#712)

Commit:
fix: tasks fragment margins (#728)

vj-codes committed:
docs: update README with app features (#717)

isabelcosta committed:
docs: update README with a little bit of docs (#737)

eraditya1024 committed:
fix: Update collapsing toolbar text size (#738)

* fix: Update collapsing toolbar text size
* fix: Shifted hardcoded text size value to dimens.xml

HaripriyaB committed:
fix: new task floating button visible now (#745)

* Fix for #744 New task floating button visible now
* Added to dimens

CodeChamp-SS committed:
docs: add badges to README (#749)

pknotfound committed:
Fix: Last item hiding behind bottom nav (#726)

SHITIZ-AGGARWAL committed:
docs: create manual test workflow-accept-relation.md (#958)

atharwa-24 committed:
fix: remove delete account feature (#1019)

Signed-off-by: atharwa-24 <atharwakharkar@gmail.com>

ashwinijha6 committed:
docs: add blog post about OSH (#999)

Hyper-Matrix committed:
feat: confirmation dialog when accepting / rejecting / deleting request (#949)

Hyper-Matrix committed:
fix: Keyboard appearance in the Signup page (#895)

priyanshuone6 committed:
fix: disorientation of the cancel button in landscape mode (#889)

ankit-saraf committed:
fix: filter users by non case sensitive inputs (#875)

CodeChamp-SS committed:
fix: centering the ui in relations tab (#819)

siddheshkothadi committed:
fix: disabled the login button until the user has filled all the credentials (#812)

DhaneshShetty committed:
fix: maladaptive about page constraints (#798)

justdvnsh committed:
tests: additional tests for signup feature (#936)

justdvnsh committed:
tests: fixed all the failing Login Activity Tests (#921)

body: Fixed the issue for #918. Fixed all the failing tests in the login activity

Anmollenka committed:
docs: add to README.md instructions on how to run UI tests (#888)

siddheshkothadi committed:
chore: add GitHub Actions that run same steps as Travis CI (#876)

Co-authored-by: Isabel Costa <11148726+isabelcosta@users.noreply.github.com>

CodeChamp-SS committed:
fix: feedback screen validation error (#991)

atharwa-24 committed:
docs: add logo to the existing badges on README (#998)

Added logos to get the info. about the repos ci ,etc system at first glance

justdvnsh committed:
test: add Logout test (#834)

Add Test For the Logout Button in Settings Activity to check whether the button is logging the user out. Fix for Issue #801

TreavVasu committed:
style: change App name to Mentorship System (#814)

justdvnsh committed:
test: add test for Change Password Fragment (#836)

* test: Added Test for Change Password Fragment - Scenario (b)
body: Added Test for the Change Password -> When the password does not match. Fixes #804

somayaj committed:
style: remove redundant app namespace (#1032)

* Removed redundant app namespace - fix for #1008
* Removed redundant android namespace - fix for #1008

epicadk committed:
fix: replace Systers with AnitaB.org in the About screen (#961)

shivamparashar165 committed:
fix: spelling mistake in SignUpActivityTest (#1045)

ankit-saraf committed:
docs: audit accessibility issues (#932)

priyanka0906 committed:
docs: fix unit test mention to UI in README (#1043)

spacemonkey1101 committed:
fix: mentorship system logo systers mention (#1012)

DhaneshShetty committed:
fix: resolve long text cut in bottom nav (#853)

DhaneshShetty committed:
feat: set condition for enabling change password btn (#837)

faznan3nazer committed:
docs: add feature list to Readme.md (#966)

maharishi-01 committed:
docs: create test workflow for reject mentorship request (#964)

Sonali-Sharma-1 committed:
feat: LoginActivity has singleTask launchMode attribute (#1037)

Rahulm2310 committed:
docs: add kotlin docs link in readme.md and contributing.md (#1067)

Sanika1999 committed:
docs: comment out description placeholder in templates (#1068)

* docs : Comment out descriptions

* Update bug_report.md

* Update PULL_REQUEST_TEMPLATE.md

devsebastian committed:
test: Request Detail Screen Espresso test (a) (#1060)

devsebastian committed:
test: "Request Detail" Screen Espresso test (b) (#1059)

darahask committed:
feat: runtime checks to signup for the email and confirm password (#954)

This adds runtime validation for the email and confirm password.
This resolves 2nd and 3rd point of issue#782.

swaraj961 committed:
docs: updated the code of conduct (#885)

Zulfa210 committed:
chore: use anitab instead of systers in the app id (#1071)

Changed application id from org.systers.mentorship to org.anitab.mentorship

varunvjha committed:
docs: update Code of Conduct (#1078)

Preshit-Joshi committed:
style: fix hardcoded strings in SignUp activity (#1076)

werainkhatri committed:
docs: guideline to resize image and table in markdown (#1082)

kartikeysaran committed:
chore: delete .travis.yml (#1084)

HeroicHitesh committed:
docs: manual test for create tasks and complete the task (#906)

Co-authored-by: Isabel Costa <11148726+isabelcosta@users.noreply.github.com>

epicadk committed:
test: fix login activity tests (#1030)

epicadk committed:
test: fix bug when user enters incorrect password (#984)

AkinchanKushwaha committed:
docs: delete .github/ISSUE_TEMPLATE folder (#1105)

epicadk committed:
feat: open about links in relevant apps (#1081)

heyanurag committed:
docs: update screenshots in README (#1017)

sharur7 committed:
docs: manual test delete mentorship relation (#941)

CodeChamp-SS committed:
fix: navigation tabs in requests page swipe properly (#1100)

maharishi-01 committed:
fix: add password check while changing passwords (#815)

CodeChamp-SS committed:
refactor: init isDone field using completedAt (#1091)

dhruvmendiratta6 committed:
fix: bottom members now visible while scrolling (#1077)

justdvnsh committed:
test: add test for Current Relation screen scenario (#835)

Add Test for Current Relation Screen Scenario (a) -> When User Has  not relation

CodeChamp-SS committed:
feat: disable signup button until user gives all the required credentials (#817)

isabelcosta committed:
chore: add all-contributors configuration (#1114)

allcontributors[bot] committed:
docs: add isabelcosta as a contributor (#1115)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

allcontributors[bot] committed:
docs: add vj-codes as a contributor (#1116)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

allcontributors[bot] committed:
docs: add devkapilbansal as a contributor (#1117)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

allcontributors[bot] committed:
docs: add epicadk as a contributor (#1118)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

allcontributors[bot] committed:
docs: add gaurivn as a contributor (#1119)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

allcontributors[bot] committed:
docs: update vj-codes as a contributor (#1120)

* docs: update README.md [skip ci]

* docs: update .all-contributorsrc [skip ci]

Co-authored-by: allcontributors[bot] <46447321+allcontributors[bot]@users.noreply.github.com>

ManishPanwar556 committed:
style: order strings in alphabetical order (#1126)

aakankshaa23 committed:
fix: match edit profile landscape and portrait mode (#1112)

unc0ded committed:
feat: better UX for successful feedback submission (#1122)

Signed-off-by: unc0ded <56927116+unc0ded@users.noreply.github.com>

TreavVasu committed:
docs: how to test using Emulator and Apk (#883)

kartikeysaran committed:
refactor: instantiate View Models using delegate (#1096)

Co-authored-by: kartikeysaran <kartikeysaran@users.noreply.github.com>

justdvnsh committed:
test: add Edit Profile screen test (#878)

Marvel999 committed:
chore: remove jcentre repository (#1127)

unc0ded committed:
chore: set up spotless for ensuring code style (#1123)

* feat: set up spotless for kotlin and xml formatting
* chore: set up github action for lint check
* style: format kotlin and xml code using spotless

niyati-n committed:
chore: add welcome bot configuration (#909)

Co-authored-by: Isabel Costa <11148726+isabelcosta@users.noreply.github.com>

epicadk committed:
chore: configure R8 for release build (#1069)

* Configure R8 for release build
* Remove Redundant Proguard rules
* Quality Assurance (QA) type of task

epicadk committed:
chore: use kotlin Scripts for gradle builds (#1062)

Signed-off-by: epicadk <adityakurkure@gmail.com>

epicadk committed:
chore: Add isabel as a code owner (#1134)

epicadk committed:
chore: update SpotlessConfig.kt (#1135)

epicadk committed:
refactor: use Coroutines Instead of rxjava (#1057)

nidhisingh-1 committed:
fix: rename welcome_bot.yml to config.yml (#1145)

Ankit-akumar committed:
feat: password validation improvement on registration (#1138)

Yash-Raj-Singh committed:
docs: add Logout Screen Page Screenshot to README.md (#1150)

Yash-Raj-Singh committed:
feat: add toast when no user is filtered by location (#1149)

kulloveth committed:
docs: remove Travis and Codacy Badge table (#1156)

Yash-Raj-Singh committed:
feat: moving docs from wiki to Docusaurus (#1154)

Acash512 committed:
fix: remove error icons from pw field so that visibility toggle is clickable (#1143)

Fr1x3 committed:
docs: add Docusaurus badge to README (#1162)

vj-codes committed:
fix: update all instances of systers to anitab in code (#1164)

AidaIssayeva committed:
docs: add the link to project roadmap (#1168)

Tanushka11 committed:
Update README.md grammatical error

samiksha-14 committed:
Fixed grammatical errors in the 'Features' part of Readme.md

vj-codes committed:
Update issue templates

Chandra-Sekhar-Bala committed:
Login successful message change 

Co-authored-by: Vaishnavi Joshi <vj.codes29@gmail.com>

abhilipsasahoo03 committed:
Added the section "Why to use MVVM Architecture?" in TechStack.md file (#1211)

Chandra-Sekhar-Bala committed:
Fixes #1209 String update  (#1213)

* Login successful message change

* Our github Link to our github repo in strings.xml file

Co-authored-by: Vaishnavi <vj.codes29@gmail.com>
